aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -224,6 +224,22 @@ Routes can be [CORS](xref:security/cors?view=aspnetcore-6.0) enabled using [CORS
 
 For more information, see <xref:security/cors?view=aspnetcore-6.0>
 
+## ValidateScopes and ValidateOnBuild
+
+<xref:Microsoft.Extensions.DependencyInjection.ServiceProviderOptions.ValidateScopes> and <xref:Microsoft.Extensions.DependencyInjection.ServiceProviderOptions.ValidateOnBuild> are enabled by default in the [Development](xref:fundamentals/environments).
+
+The following code shows `ValidateScopes` is enabled by default in development mode but disabled in release mode:
```

Comment:
Can we provide an explainer of why this is enabled in `Development` by default? We can explain that it can help catch potential memory leaks if you're attempt to resolve a scoped service as a singleton and retaining it in memory longer than the scope of the request.